game/hud/src/services/session/layoutItems/RefillAmmo.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * @Author: Mehuge (mehuge@sorcerer.co.uk)
+ * @Date: 2017-04-12 20:01:27
+ * @Last Modified by: Mehuge (mehuge@sorcerer.co.uk)
+ * @Last Modified time: 2017-04-13 15:58:56
+ */
+
+
+import { LayoutMode, Edge } from '../../../components/HUDDrag';
+import RefillAmmo from '../../../widgets/RefillAmmo';
+
+export default {
+  position: {
+    x: {
+      anchor: 9,
+      offset: 3,
+    },
+    y: {
+      anchor: 7,
+      offset: 75,
+    },
+    size: {
+      width: 120,
+      height: 35,
+    },
+    scale: 1,
+    opacity: 1,
+    visibility: true,
+    zOrder: 99,
```

Comment:
I wish there was a better way to manage the zOrder. This is already kind of annoying to manage because there's already 17 hud widgets. Maybe we should think about putting the zOrder's of each hud widget in a separate file so we can manage this a little better, especially as we add more hud widgets.